.github/workflows/dependencies/clang.sh

```diff
@@ -1,45 +1,50 @@
 #!/usr/bin/env bash
 #
-# Copyright 2024 The WarpX Community
+# Copyright 2025 The WarpX Community
 #
 # License: BSD-3-Clause-LBNL
-# Authors: Luca Fedeli
 
 set -eu -o pipefail
 
-# `man apt.conf`:
-#   Number of retries to perform. If this is non-zero APT will retry
-#   failed files the given number of times.
+# `man apt.conf`: number of retries to perform (if non-zero,
+# APT will retry failed files the given number of times).
 echo 'Acquire::Retries "3";' | sudo tee /etc/apt/apt.conf.d/80-retries
 
-# This dependency file is currently used within a docker container,
-# which does not come (among others) with wget, xz-utils, curl, git,
-# ccache, and pkg-config pre-installed.
```

Comment:
I think what was stated in this comment is not valid since we removed the Ubuntu 23.10 container in #5474.